zcash_client_backend/src/data_api.rs

```diff
@@ -899,8 +907,11 @@ pub trait WalletRead {
         Ok(HashMap::new())
```

Comment:
This is not modified by this PR; only the comment is altered. It probably isn't necessary for this to have a default implementation, but it's outside the scope of this PR to remove it.

Reply:
This is a default implementation; it's necessary because if a downstream the `transparent-inputs` feature is enabled (say by some transitive third-party dependency) we don't want compilation of downstream crates to break if they haven't implemented this trait method.